README.md

```diff
@@ -0,0 +1,198 @@
+# Hello-World-Microverse-Week1
+
+<a name="readme-top"></a>
+
+
+
+
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World Project] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences. This is my first project in Microverse
+
+**[Hello World Project]** 
+
+## 🛠 Built With <a name="built-with"> HTML and CSS</a>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Header with a green color]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Text editor, run commands and browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/AFakhriddinov/Hello-World-Microverse-Week1
+```
+--->
+
```

Comment:
- [ ] Great job on showing how to set up your project. However, can you please uncomment the commands to run to clone your project so that people can se them? To uncomment the commands please remove `line 75` and `line 82` and remove `line 76` as well because you don't need it any more. 👌😊